docs/Images.md

```diff
@@ -40,6 +40,25 @@ pdf.image("imgB.jpg", h=pdf.eph, w=pdf.epw/2, x=pdf.epw/2)  # full page height,
 pdf.output("side-by-side.pdf")
 ```
 
+### Blending images ###
+
+You can control the color blending mode of overlapping images.
+Valid values for `blend_mode` are `Normal`, `Multiply`, `Screen`, `Overlay`, `Darken`, `Lighten`, `ColorDodge`,
+`ColorBurn`, `HardLight`, `SoftLight`, `Difference`, `Exclusion`, `Hue`, `Saturation`, `Color` and `Luminosity`.
+
+```python
+from fpdf import FPDF
+
+pdf = FPDF()
+pdf.add_page()
+pdf.image("imgA.png", ...)
+with pdf.local_context(blend_mode="ColorBurn"):
```

Comment:
> do you prefer users to supply the values as strings?

Both enum values & string values are allowed. The later ones will be [automatically coerced to enums](https://github.com/PyFPDF/fpdf2/blob/master/fpdf/drawing.py#L1018)

> Since this enum is now also used outside of drawing.py, would it make sense to move it to enums.py?

Good idea!